src/UI/Component/Signal.php

```diff
@@ -12,4 +12,18 @@
  */
 interface Signal {
 
+	/**
+	 * Get the ID of this signal
+	 *
+	 * @return string
+	 */
+	public function getId();
```

Comment:
In linked example, I could send the ID of the replaceSignal implicitly via GET. However, I see the following problems:
* How to name the GET parameters if multiple signals are involved? Maybe a developer needs multiple replaceSignals, e.g. two from different Popovers and one from a Modal. By encoding and sending the signal IDs explicitly, the developer is responsible to avoid naming conflicts for the GET parameters and is able to identify the correct component based on the GET parameter.
* This can be seen as a more general problem: When working with async requests to replace/exchange data of components, how do we identify a component that was rendered with the synced (initial) request.